index.html

```diff
@@ -1,33 +1,35 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 <head>
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<title></title>
+	<title>lass</title>
 	<script src="https://kit.fontawesome.com/b8019b4db4.js" crossorigin="anonymous"></script>
 	<link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
 	<header class="container">
-		<div id="logo">My Logo</div>
+		<div id="logo"><a href="#">My Logo</a></div>
 		<nav>
-			<button><i class="fas fa-bars" id="bar"></i></button>
+			<button type="button"><i class="fas fa-bars" id="bar"></i></button>
 		</nav>
 	</header>
 
 	<main class="container">
 		<section id="headline">
-			<h2>I’m Mariane
-				Glad to see you!</h2>
+			<h1>I’m Mariane
+				Glad to see you!</h1>
 
 			<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
 
 			<div id="connect">Let's Connect</div>
```

Comment:
- [ ] OPTIONAL: Please use the provided font family which is `Poppins`. You can see design specifications like colors, sizes etc., on the right side on Figma, you only need to click on items: 
<img width="663" alt="Screenshot 2022-03-15 at 21 46 36" src="https://user-images.githubusercontent.com/77038610/158439681-9abb67fc-aac5-45d3-895c-9e7739f8c2a1.png">